Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/css/search.css

```diff
@@ -45,4 +45,18 @@
 .search-candidates .selected {
     background-color: var(--blue);
     color: white;
+}
+
+.next-regress-bar {
+    background-color: var(--blurBackgroundColor);
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    width:100%;
+    z-index:10
+}
+
+@media (prefers-color-scheme: dark) {
+    .next-regress-bar {
+        background-color: var(--blurBackgroundColorDark);
```

Comment:
This background color doesn't match the section's background color (should be `grayDarker`, is `blurBackgroundColorDark`).

Reply:
There is a reason for that, when you have a lot of timelines, this search bar can hover on the top. so you don't need to scroll back and hit the button. it will be better to give this a different color to inform user this is a float bar.

Reply:
Why does this have a background at all? With only a single button, I think I'd rather have this be transparent.